test/run

```diff
@@ -57,25 +68,54 @@ run_s2i_build_proxy() {
   ct_s2i_build_as_df file://${test_dir}/test-hw ${IMAGE_NAME} ${IMAGE_NAME}-testhw ${s2i_args} $(ct_build_s2i_npm_variables) -e HTTP_PROXY=$1 -e http_proxy=$1 -e HTTPS_PROXY=$2 -e https_proxy=$2
 }
 
+run_s2i_build_express() {
+  ct_s2i_build_as_df \
+    "file://${test_dir}/express.js" "${IMAGE_NAME}" "${IMAGE_NAME}-express" \
+    ${s2i_args} \
+    $(ct_build_s2i_npm_variables) -e NODE_ENV=development
+}
+
+prepare_dummy_git_repo() {
+  git init
+  for key in "${!gitconfig[@]}"; do
+    git config --local "$key" "${gitconfig[$key]}"
+  done
+  git add --all
+  git commit -m "Sample commit"
+}
+
+prepare_express_repo() {
+  git clone \
+    --config advice.detachedHead="false" \
+    --branch "${EXPRESS_REVISION}" --depth 1 \
+    'https://github.com/expressjs/express.git' "$1"
+  for key in "${!gitconfig[@]}"; do
+    git -C "$1" config --local "$key" "${gitconfig[$key]}"
```

Comment:
Option `-C` does not exist in our testing `git` version. I would prefer to switch into `--work-dir` option.
```suggestion
    git --work-dir "$1" config --local "$key" "${gitconfig[$key]}"
```
https://git-scm.com/docs/git#Documentation/git.txt---work-treeltpathgt

Reply:
See error from tests:

https://gist.github.com/rhscl-bot/c1714c8f217a97bca53924c6c165a626#file-s2i-nodejs-container-179-sh-L3443

```bash
Running express.js test suite
Cloning into '/root/sources/test/express.js'...
Unknown option: -C
usage: git [--version] [--help] [-c name=value]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p|--paginate|--no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           <command> [<args>]
```

Reply:
Yeah, I noticed and used pushd/popd to change to the git root to avoid messing with extra CLI arguments… and then GitHub had a hiccup and the commit remained unpushed on my machine :slightly_smiling_face: Pushing it now.